docs/about/release-notes.rst

```diff
@@ -49,12 +49,14 @@ v22.11.0 (November 2022)
 Features
 ~~~~~~~~
 
-* Add :py:func:`scipp.elemwise_func` for defining custom transformations of input variables based on a Python function compiled via ``numba.cfunc`` `#2886 <https://github.com/scipp/scipp/pull/2886>`_.
+* Added :py:func:`scipp.elemwise_func` for defining custom transformations of input variables based on a Python function compiled via ``numba.cfunc`` `#2886 <https://github.com/scipp/scipp/pull/2886>`_.
+* Added support for arbitrary unit, degrees Celsius and other special units `#2931 <https://github.com/scipp/scipp/pull/2931>`_.
 
 Breaking changes
 ~~~~~~~~~~~~~~~~
 
 * The SciPy wrappers ``integrate``, ``interpolate``, ``ndimage``, ``optimize``, and ``signal`` were moved into the :py:mod:`scipp.scipy` submodule `#2881 <https://github.com/scipp/scipp/pull/2881>`_.
+* The HDF5 format changed to accommodate more units, old files can still be loaded but new ones cannot be loaded with an old version of Scipp `#2931 <https://github.com/scipp/scipp/pull/2931>`_.
```

Comment:
You edited this into the already releases version.

Reply:
Oh shit. Moving...